enterprise-suite/es-grafana/cassandra-exporter.json

```diff
@@ -1,7 +1,7 @@
 [
-  {"graphName":"Storage", "promQL":["irate(cassandra_storage_load{workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Reads", "promQL":["irate(cassandra_clientrequest_latency{clientrequest='Read', workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Writes", "promQL":["irate(cassandra_clientrequest_latency{clientrequest='Write', workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Read Latency", "promQL":["irate(cassandra_clientrequest_totallatency{clientrequest='Read', workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Write Latency", "promQL":["irate(cassandra_clientrequest_totallatency{clientrequest='Write', workload=\"$workloadId\"}[1m])"]}
+  {"graphName":"Storage", "promQL":["cassandra_storage_load{workload=\"$workloadId\"}"]},
+  {"graphName":"Read Latency", "promQL":["cassandra_clientrequest_latency{clientrequest='Read', workload=\"$workloadId\"}"]},
+  {"graphName":"Write Latency", "promQL":["cassandra_clientrequest_latency{clientrequest='Write', workload=\"$workloadId\"}"]},
+  {"graphName":"Total Read Latency", "promQL":["cassandra_clientrequest_totallatency{clientrequest='Read', workload=\"$workloadId\"}"]},
```

Comment:
These `totallatency` must be differentiated, monotonic graphs are too hard to read.  These are lifetime sums of latency.  These should be irate and "Latency per Second"